internal/api/db/db.go

```diff
@@ -0,0 +1,20 @@
+package db
+
+import (
+	"omshub/core-api/internal/api/db/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func Init() *gorm.DB {
+	dsn := "host=localhost user=postgres password=postgres dbname=postgres port=5432 sslmode=disable"
+	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
+	if err != nil {
+		panic("failed to connect database")
+	}
+
+	_ = db.AutoMigrate(&models.Review{})
```

Comment:
Let's capture and log this error

Reply:
pass out `err`, if not nil will log a warning